src/CallHandler.js

```diff
@@ -444,8 +445,8 @@ function _startCallApp(roomId, type) {
         'email=$matrix_user_id',
     ].join('&');
     const widgetUrl = (
-        'https://scalar.vector.im/api/widgets' +
-        '/jitsi.html?' +
+        SdkConfig.get().integrations_rest_url +
+        '/widgets/jitsi.html?' +
```

Comment:
fwiw this breaks in Dimension given it expects an entirely different format. Is it possible to just have the entire URL configurable? 

At least until https://github.com/matrix-org/matrix-doc/issues/1286 is a thing. It would also be nice if both Scalar and Dimension can agree on the parameters being used, as per the `m.jitsi` widget declaration in https://docs.google.com/document/d/1uPF7XWY_dXTKVKV7jZQ2KmsI19wn9-kFRgQ1tFQP7wQ/edit#